charts/linkerd2/values.yaml

```diff
@@ -31,6 +31,8 @@ global:
   # proxy configuration
   proxy:
     enableExternalProfiles: false
+    outboundConnectTimeout: 1000ms
+    inboundConnectTimeout: 100ms
```

Comment:
@mmiller1 This change to `values.yml` requires an update to the [values_test.go](https://github.com/mmiller1/linkerd2/blob/master/pkg/charts/linkerd2/values_test.go#L53) to include the two new values in the default configuration.

Reply:
ah thanks, added these values.